packages/babel-compat-data/scripts/data/plugin-features.js

```diff
@@ -70,36 +64,66 @@ module.exports = {
       "destructuring, parameters / defaults, arrow function",
     ],
   },
-  "transform-property-literals":
-    "Object/array literal extensions / Reserved words as property names",
-  "transform-regenerator": {
-    features: ["generators"],
+  "transform-destructuring": {
+    features: ["destructuring, assignment", "destructuring, declarations"],
   },
-  "transform-reserved-words": "Miscellaneous / Unreserved words",
-  "transform-shorthand-properties": {
-    features: ["object literal extensions / shorthand properties"],
+  "transform-block-scoping": {
+    features: ["const", "let"],
   },
-  "transform-spread": {
-    features: "spread syntax for iterable objects",
+  "transform-typeof-symbol": {
+    features: ["Symbol / typeof support"],
   },
-  "transform-sticky-regex": {
-    features: [
-      'RegExp "y" and "u" flags / "y" flag, lastIndex',
-      'RegExp "y" and "u" flags / "y" flag',
-    ],
+  "transform-new-target": {
+    features: ["new.target"],
   },
-  "transform-template-literals": {
-    features: ["template literals"],
+  "transform-regenerator": {
+    features: ["generators"],
   },
-  "transform-typeof-symbol": {
-    features: ["Symbol / typeof support"],
+};
+
+const es2016 = {
+  "transform-exponentiation-operator": {
+    features: ["exponentiation (**) operator"],
   },
-  "transform-unicode-regex": {
-    features: [
-      'RegExp "y" and "u" flags / "u" flag, case folding',
-      'RegExp "y" and "u" flags / "u" flag, Unicode code point escapes',
-      'RegExp "y" and "u" flags / "u" flag, non-BMP Unicode characters',
-      'RegExp "y" and "u" flags / "u" flag',
-    ],
+};
+
+const es2017 = {
+  "transform-async-to-generator": {
+    features: ["async functions"],
   },
 };
+
+const es2018 = {
+  "proposal-async-generator-functions": "Asynchronous Iterators",
+  "proposal-object-rest-spread": "object rest/spread properties",
+
+  // We want to apply this prior to unicode regex so that "." and "u"
```

Comment:
I don't think the order of regex-related plugins still matters since they are essentially feature flags now, the `regexpu-core` should only run once for every applicable regex.